cloudsync/tests/test_utils.py

```diff
@@ -23,10 +23,8 @@ def test_debug_args():
 
 @pytest.mark.manual
 def test_multiline():
-    from cloudsync.utils import disable_log_multiline
     log.error("indented line1\n<--- weird pytest indentation")
-    with disable_log_multiline():
-        log.error("not indented line1\n<--- right up against the edge of the terminal")
+    log.error("not indented line1\n<--- right up against the edge of the terminal")
```

Comment:
I think you can get rid of the whole test_multiline method, I don't think there's a reason for this test anymore